src/components/FixedGrid/Grid.js

```diff
@@ -226,11 +227,11 @@ function Grid(props) {
       ref={props.fixedRef}
     >
       {pages.length > 0 ? (
-        pages.map((pageItems, i) => (
+        pages.map((_, i) => (
```

Comment:
Instead of make a map of the `pages` we can directly use the new `memoizedGridState` to avoid this complex logic to render the correct gridState